Index: webkit-efl-svn-r98964/Source/WebKit/CMakeLists.txt
===================================================================
--- webkit-efl-svn-r98964.orig/Source/WebKit/CMakeLists.txt
+++ webkit-efl-svn-r98964/Source/WebKit/CMakeLists.txt
@@ -70,7 +70,7 @@ ADD_DEFINITIONS(-DBUILDING_WebKit)
 INCLUDE_DIRECTORIES(${WebKit_INCLUDE_DIRECTORIES})
 ADD_LIBRARY(${WebKit_LIBRARY_NAME} ${WebKit_LIBRARY_TYPE} ${WebKit_SOURCES})
 ADD_DEPENDENCIES(${WebKit_LIBRARY_NAME} ${WebCore_LIBRARY_NAME})
-TARGET_LINK_LIBRARIES(${WebKit_LIBRARY_NAME} ${WebKit_LIBRARIES})
+TARGET_LINK_LIBRARIES(${WebKit_LIBRARY_NAME} ${WebKit_LIBRARIES} ${Gthread_LIBRARIES})
 
 IF (WebKit_LINK_FLAGS)
     ADD_TARGET_PROPERTIES(${WebKit_LIBRARY_NAME} LINK_FLAGS "${WebKit_LINK_FLAGS}")
